daemon/algod/api/server/v2/utils.go

```diff
@@ -414,9 +414,88 @@ func convertTxnResult(txnResult simulation.TxnResult) PreEncodedSimulateTxnResul
 		AppBudgetConsumed:      numOrNil(txnResult.AppBudgetConsumed),
 		LogicSigBudgetConsumed: numOrNil(txnResult.LogicSigBudgetConsumed),
 		TransactionTrace:       convertTxnTrace(txnResult.Trace),
+		UnnamedResources:       convertUnnamedResourceAssignment(txnResult.UnnamedResources),
 	}
 }
 
+func convertUnnamedResourceAssignment(assignment *simulation.ResourceAssignment) *model.SimulationUnnamedResourceAssignment {
+	if assignment == nil {
+		return nil
+	}
+	converted := model.SimulationUnnamedResourceAssignment{
+		MaxTotalRefs: uint64(assignment.MaxTotalRefs),
+		MaxAccounts:  uint64(assignment.MaxAccounts),
+		MaxAssets:    uint64(assignment.MaxAssets),
+		MaxApps:      uint64(assignment.MaxApps),
+		MaxBoxes:     uint64(assignment.MaxBoxes),
+	}
+	if len(assignment.Accounts) != 0 {
+		accountStrings := make([]string, 0, len(assignment.Accounts))
+		for account := range assignment.Accounts {
+			accountStrings = append(accountStrings, account.String())
+		}
+		converted.Accounts = &accountStrings
+	}
+	if len(assignment.Assets) != 0 {
+		assets := make([]uint64, 0, len(assignment.Assets))
+		for asset := range assignment.Assets {
+			assets = append(assets, uint64(asset))
+		}
+		converted.Assets = &assets
+	}
```

Comment:
Added generic function for this in f340fc6b210ebf0fb82e9319451ff1a036965df3. It's slightly more limited because this code involves map keys, but I think still useful